pkg/walker.go

```diff
@@ -80,6 +82,17 @@ func (w *Walker) doWalk(ctx context.Context, p *Parser, dir string, visitor Walk
 
 		if file.IsDir() {
 			if w.Recursive {
+				nextFiles, err := ioutil.ReadDir(path)
+				if err != nil {
+					log.Err(err).Msgf("Error reading directory")
+					return
+				}
+				for _, nextFile := range nextFiles {
+					if nextFile.Name() == ".mockery_skip" {
+						continue OUTER
```

Comment:
I tend to really avoid labels if at all possible. Actually, I think we can `stat` for `.mockery_skip` at the top of `doWalk` and simply return if it exists.

Shameless plug but you might want to look into https://pkg.go.dev/github.com/chigopher/pathlib and then you can just do `pathlib.Path(dir).Joinpath(".mockery_skip").Exists()` to check.